### DISCUSSION TITLE
Meshtastic an option?

### DISCUSSION THREAD

**Discussion by IngenioCraft** [ludwich66/Quansheng_UV-K5_Wiki#29] [Ideas]
I am inspired by the digital SMS on these otherwise analogue radios…

 Is Meshtastic protocol a possibility on this hardware?  It would be amazing if possible !!

[h](https://meshtastic.org)

**Comment by giulioPerissi** [comment]
Unfortunately not, Meshtastic is based on the LoRa modulation (a refined FSK mode, in fact most LoRa transceiver can also do FSK), this radio can only work in FM (AM with some software trick), so NO you can't.
There will be the possibility by modifying: the PCB to insert the LoRa transceiver, the firmware,  the RF circuit or using another antenna, and probably also the case.
